services/brig/test/integration/Main.hs

```diff
@@ -70,7 +71,13 @@ runTests iConf bConf = do
     teamApis    <- Team.tests bConf mg b c g
     turnApi     <- TURN.tests mg b turnFile
 
-    defaultMain $ testGroup "Brig API Integration"
+    -- you can now do this (see <https://github.com/feuerbach/tasty#patterns>):
+    -- `WIRE_TASTY_PATTERN='$NF == "post /register - 201 + no email"' make integration`
+    otherArgs' <- lookup "WIRE_TASTY_PATTERN" <$> getEnvironment >>= pure . \case
```

Comment:
This seems to be the same as `lookupEnv "WIRE_TASTY_PATTERN" <&> \case ...`

Reply:
cool.  i've been meaning to find that thing, thanks!  will fix.